examples/basic/src/components/Overlay.tsx

```diff
@@ -216,8 +217,17 @@ const _Overlay = forwardRef<VideoRef, Props>((props, ref) => {
   };
 
   const onResizeModeSelected = (value: MultiValueControlPropType) => {
-    if (typeof value === 'object') {
-      setResizeMode(value);
+    if (typeof value === 'string') {
+      switch (value) {
+        case ResizeMode.CONTAIN:
```

Comment:
Following previous comment, I think you can use 'as' to simplify this function, can you have a look please?